packages/editor/src/client/blocks/blockMap.tsx

```diff
@@ -56,6 +58,30 @@ export const BlockMap: BlockMapForValue<BlockValue> = {
     icon: MaterialIconCollections
   },
 
+  [BlockType.MapLeaflet]: {
+    field: props => <MapLeafletBlock {...props} />,
+    defaultValue: {
+      centerLat: 46.948,
```

Comment:
Explain the default values.

Reply:
Default value is Bern (capital). Do you mean explaining it here or explain it with a comment in the code?